frontend/src/metabase/static-viz/components/Legend/Legend.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Group } from "@visx/group";
+import { Text } from "metabase/static-viz/components/Text";
+import { PositionedLegendItem } from "./types";
+
+import { LEGEND_CIRCLE_MARGIN_RIGHT, LEGEND_CIRCLE_SIZE } from "./constants";
+
+type LegendProps = {
+  top: number;
+  left: number;
+  lineHeight: number;
+  fontSize: number;
+  fontWeight: number;
+  items: PositionedLegendItem[];
+};
+
+export const Legend = ({
+  top,
+  left,
+  lineHeight,
+  fontSize,
+  fontWeight,
+  items,
+}: LegendProps) => {
+  return (
+    <Group left={left} top={top}>
+      {items.map((item, index) => {
+        const { name, color, left, top } = item;
+        const radius = LEGEND_CIRCLE_SIZE / 2;
+        const textX = LEGEND_CIRCLE_SIZE + LEGEND_CIRCLE_MARGIN_RIGHT;
+
+        return (
+          <Group left={left} top={top} key={name}>
+            <circle fill={color} r={radius} cx={radius} cy={radius} />
+            <Text
+              textAnchor="start"
+              verticalAnchor="start"
+              x={textX}
+              fontWeight={fontWeight}
+              lineHeight={lineHeight}
```

Comment:
https://github.com/airbnb/visx/blob/04f191f3cf8a0fcab569c50cd6ec38f247a2ec7f/packages/visx-text/src/Text.tsx#L34

`lineHeight` doesn't seem to do anything from my testing, so I dug into it, and apparently, this only affects `<Text />` that spans multiple lines (have `width` i.e. max-width set.)

You can try playing around with that here. https://airbnb.io/visx/text

If it doesn't do anything maybe we shouldn't leave it here, this gives the impression that we need `lineHeight` here.